docs/extensions/extensions_example.md

```diff
@@ -185,14 +188,14 @@ more than one version in our lockfile.
 
 For our example, the output looks like this:
 
-```
+```console
```

Comment:
What language is this even supposed to be? I don't think we need syntax highlighting for every little code block.

Reply:
It is meant to represent code formatting for a shell session. [GitHub markdown](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) supports this code fence "language" via [linguist](https://github.com/github/linguist/blob/59492635f1f7c156289aa38be14f56037a1c6784/lib/linguist/languages.yml#L6090), but apparently [Readme.com does not](https://rdmd.readme.io/docs/code-blocks).

The only reason for ensuring a language is specified for all fenced code blocks is to make [markdownlint (rule MD040)](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md040) happy...which is enabled in my IDE and may be included in future QA checks across Phylum's repos. The rule's rationale is:

> Specifying a language improves content rendering by using the correct syntax highlighting for code.

That said, it may not make sense for _every_ code block, but doing so makes it clear that it was not accidentally forgotten.

Reply:
> That said, it may not make sense for every code block, but doing so makes it clear that it was not accidentally forgotten.

That's precisely the issue though, just making stuff colorful with no rhyme or reason doesn't actually improve anything. For normal text the `console` thing will just color everything blue, even on GitHub. That's not actually an improvement, quite the opposite actually.

Having just plain text codeblocks shouldn't be an issue.